client/src/components/ItemCard.tsx

```diff
@@ -51,17 +62,33 @@ export const ItemCard = ({ item, onClick }: Props) => {
         })}
         onClick={handleClick}
       >
-        <Image
-          priority
-          src={item.image}
-          alt={`select ${
-            "first_name" in item
-              ? `${item.first_name} ${item.last_name}`
-              : item.name
-          }`}
-          width={200}
-          height={280}
-        />
+        {isInstrument ? (
+          // render players
+          <Image
+            priority
+            src={item.image}
+            alt={`select ${
+              "first_name" in item
+                ? `${item.first_name} ${item.last_name}`
+                : item.name
+            }`}
+            width={200}
+            height={200}
+          />
+        ) : (
+          // render instruments
+          <Image
+            priority
+            src={item.image}
+            alt={`select ${
+              "first_name" in item
+                ? `${item.first_name} ${item.last_name}`
```

Comment:
probably just `item.name` for this one I think, since instruments don't typically have last names :)

Reply:
It was actually just `item` 

` alt={`select ${item}`}`